doc/rados/configuration/ceph-conf.rst

```diff
@@ -620,84 +623,84 @@ Override values can be set in two ways:
       ceph daemon osd.4 config set debug_osd 20
 
 .. note:: In the output of the ``ceph config show`` command, these temporary
-   values are shown with a source of ``override``.
+   values are shown to have a source of ``override``.
 
 
 Viewing runtime settings
 ========================
 
-You can see the current options set for a running daemon with the ``ceph config show`` command.  For example:
+You can see the current settings specified for a running daemon with the ``ceph
+config show`` command. For example, to see the (non-default) settings for the
+daemon ``osd.0``, run the following command:
 
 .. prompt:: bash $
 
    ceph config show osd.0
 
-will show you the (non-default) options for that daemon.  You can also look at a specific option with:
+To see a specific setting, run the following command:
 
 .. prompt:: bash $
 
    ceph config show osd.0 debug_osd
 
-or view all options (even those with default values) with:
+To see all settings (even those with default values), run the following
+command:
 
 .. prompt:: bash $
 
    ceph config show-with-defaults osd.0
 
-You can also observe settings for a running daemon by connecting to it from the local host via the admin socket.  For example:
+You can see all settings for a daemon that is currently running by connecting
+to it from the local host via the admin socket. For example, to dump all
+current settings, run the following command:
 
 .. prompt:: bash $
 
    ceph daemon osd.0 config show
 
-will dump all current settings:
+To see non-default settings and to see where each value came from (for example,
+a config file, the monitor, or an override), run the following command:
 
 .. prompt:: bash $
 
    ceph daemon osd.0 config diff
 
-will show only non-default settings (as well as where the value came from: a config file, the monitor, an override, etc.), and:
+To see the value of a single setting, run the following command:
 
 .. prompt:: bash $
 
    ceph daemon osd.0 config get debug_osd
 
-will report the value of a single option.
 
-
-
-Changes since Nautilus
-======================
+Changes introduced in Octopus
+=============================
 
 The Octopus release changed the way the configuration file is parsed.
 These changes are as follows:
 
-- Repeated configuration options are allowed, and no warnings will be printed.
-  The value of the last one is used, which means that the setting last in the file
-  is the one that takes effect. Before this change, we would print warning messages
-  when lines with duplicated options were encountered, like::
+- Repeated configuration options are allowed, and no warnings will be
+  displayed.  The value of the last option is used, which means that the
```

Comment:
s/The value of the last option is used, which//

displayed.  This means that the ..

Reply:
Accepted.